cylc/flow/scripts/cylc.py

```diff
@@ -54,12 +54,14 @@ def get_version(long=False):
   {get_version(True)}
 
 Usage:
+  $ cylc help license             # view the Cylc license (GPL-3.0)
   $ cylc help all                 # list all commands
-  $ cylc validate FLOW            # validate a workflow configuration
-  $ cylc play FLOW                # run/resume a workflow
+  $ cylc validate <workflow>      # validate a workflow configuration
+  $ cylc install <workflow>       # install a workflow
+  $ cylc play <workflow>          # run/resume a workflow
   $ cylc scan                     # list all running workflows (by default)
-  $ cylc tui FLOW                 # view a running workflow in the terminal
-  $ cylc stop FLOW                # stop a running workflow
+  $ cylc tui <workflow>           # view a running workflow in the terminal
```

Comment:
```suggestion
  $ cylc tui <workflow>           # view and interact with a running workflow in the terminal
```

Reply:
This was too long so went with "view/control workflows in the terminal".